modules/ROOT/partials/configuration/advanced-typography.adoc

```diff
@@ -0,0 +1,216 @@
+[[typography_default_lang]]
+== `+typography_default_lang+`
+
+This option gives the user the ability to pre-set a default typography language within the {productname} editor. By setting up the `typography_default_lang`, when the user opens the {productname} editor for the first time, it will automatically default to the new default language applied. If this default language is subesequently changed by the user, the new default will be saved and persistent between documents.
+
+By setting up the `typography_default_lang`, each time the user opens the {productname} editor, it will automatically default to the new default language applied.
+
+=== Example: Using `typography_default_langs`
+
+[source,js,subs="attributes+"]
+----
+tinymce.init({
+  selector: "textarea",  // change this value according to your html
+  menu: { tools: { title: 'Tools', items: 'code wordcount typography' }},
+  plugins: "typography",
+  toolbar: "typography",
+  typography_default_lang: "en-US", // Define the default language here, example: "en-US"
+});
+----
+
+[[typography_ignore]]
+== `+typography_ignore+`
+
+This option tells the {pluginname} plugin to ignore content within specified HTML tags or within specified HTML tags that have specified attributes.
+
+If an HTML tag or an HTML tag with a specified attribute is added to the `typography_ignore` array, the Typograhy plugin’s rules are not applied to material set within that tag.
+
+*Type:* `+Array+`
+
+*Possible values:* `[ '<html-selector>', 'span["<html-selector>"]' ]`
+
+*Default value:* none
+
+=== Example: Using `typography_ignore`
+
+[source,js,subs="attributes+"]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: '{plugincode}',
+  toolbar: 'typography',
+  typography_default_lang: [ "en-US", "de", "es" ], // Required to set specific typography language rules.
```

Comment:
```suggestion
  typography_default_lang: 'en-US', 
```

Reply:
According to the typography code, this option only accepts a string as its value. Also, it is given an array of strings which is incorrect in other examples,